packages/dart_frog_cli/CONTRIBUTING.md

```diff
@@ -0,0 +1,134 @@
+# 🦄 Contributing to Dart Frog CLI
+
+First of all, thank you for taking the time to contribute! 🎉👍 Before you do, please carefully read this guide.
+
+## Understanding Dart Frog CLI
+
+The Dart Frog CLI is a [`CommandRunner`](https://pub.dev/documentation/args/latest/command_runner/CommandRunner-class.html) that exposes four visible commands:
+
+- build: Creates a production build by using the [`dart_frog_prod_server`](../../bricks/dart_frog_prod_server/) mason template to generate files.
+- create: Creates a new Dart Frog app by using the [`dart_frog_new`](../../bricks/dart_frog_new/) mason template to generate the files.
+- dev: Runs a local development server at a given port and listens to file changes generating files using the [`dart_frog_dev_server`](../../bricks/dart_frog_dev_server/) mason template to generate file
```

Comment:
```suggestion
- `build`: Creates a production build by using the [`dart_frog_prod_server`](../../bricks/dart_frog_prod_server/) mason template to generate files.
- `create`: Creates a new Dart Frog app by using the [`dart_frog_new`](../../bricks/dart_frog_new/) mason template to generate the files.
- `dev`: Runs a local development server at a given port and listens to file changes generating files using the [`dart_frog_dev_server`](../../bricks/dart_frog_dev_server/) mason template to generate file
```

for `dev`: should it be "...and listens to file changes using" or "...and listens to generating files using"; seems like it shouldn't be both "...and listens to file changes generating files using"